.github/repos/settings.yaml

```diff
@@ -25,6 +25,8 @@ shared-config: &shared-config
     squash-merge: true
     merge-commit: false
     rebase-merge: false
+  squash_merge_commit_title: PR_TITLE
+  squash_merge_commit_message: PR_BODY
```

Comment:
Are you sure this will work without https://github.com/PyGithub/PyGithub/issues/2403 ?
P.S. I would like to set them through `reposettings.py` too :-(